src/components/Text/Text.tsx

```diff
@@ -56,15 +50,26 @@ export const Text = forwardRef<TextPropTypes, 'p'>(({ components, translation, .
             return cloneElement(components[key], { key: index })
           }
 
-          const tagContentMatch = part.match(/<([^>]+)>(.*?)<\/\1>/)
-          if (tagContentMatch) {
-            const [, key, content] = tagContentMatch
+          const tagMatches = part.match(/<([^>]+)>(.*?)<\/\1>/g)
+          if (tagMatches) {
+            const pieces = part.split(/(<[^>]+>.*?<\/[^>]+>)/g)
 
-            if (!components?.[key]) return <span key={index}>{part}</span>
+            return pieces.map((piece, pieceIndex) => {
+              if (!piece) return null
+
+              const tagContentMatch = piece.match(/<([^>]+)>(.*?)<\/\1>/)
+
+              if (tagContentMatch) {
+                const [, key, content] = tagContentMatch
+                if (!components?.[key]) return <span key={`${index}-${pieceIndex}`}>{piece}</span>
+
+                return cloneElement(components[key], {
+                  key: `${index}-${pieceIndex}`,
+                  children: content,
+                })
+              }
 
-            return cloneElement(components[key], {
-              key: index,
-              children: content,
+              return piece ? <Fragment key={`${index}-${pieceIndex}`}>{piece}</Fragment> : null
```

Comment:
redundant with the above early return on `!piece`

Reply:
Fixed in [7f23df6](https://github.com/shapeshift/web/pull/9102/commits/7f23df658b14b70321e5607505c40a9b3f62b2ed)